schema/errors.go

```diff
@@ -247,6 +247,50 @@ func NewNoWebAuthnRegistered() error {
 	})
 }
 
+func NewHookValidationError(instancePtr, message string, messages text.Messages) *ValidationError {
+	return &ValidationError{
```

Comment:
Right. This one is a bit special since it is wrapped by `ValidationListError`. I've added `errors.Wrap()` to `NewValidationListError()` but I don't think it will be good to wrap those additionally.